app/views/dashboard/_node_meta.html.erb

```diff
@@ -4,7 +4,11 @@
 <%= time_ago_in_words(node.created_at) %> ago
 <% node = node.parent if node.is_a?(DrupalNodeRevision) %>
 <span class="">
-  <% if node.comment_count > 0 %>| <a href="<%= node.path %>#comments"><i class="fa fa-comment-o"></i> <%= node.comment_count %></a><% end %>
+  <% if params[:controller] == 'questions' %>
+    <% if node.answers.length > 0 %>| <a href="<%= node.path(:question) %>#answers"><span class="glyphicon glyphicon-comment" style="color: #DAA583;"></span> <%= node.answers.length %></a><% end %>
```

Comment:
Yes actually I wanted to use different icons for showing answer count and comments count. The bootstrap glyphicon seemed apt for answer counts.